src/shared/containers/Refile/RefileErrorsForm.jsx

```diff
@@ -50,7 +49,11 @@ class RefileErrorsForm extends Component {
     const currentState = state;
 
     if (typeof currentState.fieldErrors === 'object' && isEmpty(currentState.fieldErrors[field])) {
-      currentState.fieldErrors[field] = errorString;
+      const currentFieldErrorsState = assign(currentState.fieldErrors, { [field]: errorString });
```

Comment:
Matter of interest, could we do this with the following? Again, not a big deal :-)

```
const currentFieldErrorsState = {...currentState.fieldErrors, [field]: errorString };
```

Reply:
We can! Thanks for telling me this. I just tested it. Will update it.